src/res/translation/translation_de_DE.ts

```diff
@@ -1136,7 +1137,7 @@
     <message>
         <location filename="../../clientdlgbase.ui" line="584"/>
         <source>Set up your audio, connect to a server and start jamming!</source>
-        <translation type="unfinished"></translation>
+        <translation>Stelle bitte das Audiointerface ein , verbinde Dich mit einem Server und beginne mit dem Jammen !</translation>
```

Comment:
I think it's not only related to setting up an audio-interface. What about "Richte deine Soundkarte ein, verbinde Dich mit einem Server und beginne zu Jammen!"?